src/main/java/net/earthcomputer/clientcommands/render/Renderer.java

```diff
@@ -0,0 +1,43 @@
+package net.earthcomputer.clientcommands.render;
+
+
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Vec3d;
+
+/**
+ * Copyright (c) 2020 KaptainWutax
+ */
+public abstract class Renderer {
+
+    protected MinecraftClient mc = MinecraftClient.getInstance();
+
+    /**
+     * The life in number of ticks
+     * Nothing
+     */
+    protected int life = -1;
+    protected long prevTick;
+
+    public void render() {
+        if (this.mc.world.getTime() != this.prevTick && this.life > 0) {
+            this.life--;
+            this.prevTick = this.mc.world.getTime();
+        }
```

Comment:
```suggestion
        if (this.life > 0) {
            this.life -= this.mc.world.getTime() - this.prevTick;
        }
```
Fixes the situation where there are multiple ticks per frame.